notebook2/notebook/notebook.py

```diff
@@ -12,6 +12,7 @@
 import sass
 from authlib.flask.client import OAuth
 from urllib.parse import urlencode
+from functools import wraps
 
 import kubernetes as kube
 
```

Comment:
it's outside the diff, but can you delete `PASSWORD` since it's unused now?

Reply:
PASSWORD is still used, for the workshop password.